src/main/java/seedu/address/model/property/Property.java

```diff
@@ -64,9 +64,7 @@ public boolean isSameProperty(Property otherProperty) {
         }
 
         return otherProperty != null
-                && otherProperty.getName().equals(getName())
```

Comment:
Change does not match Javadocs. Missing && otherProperty.getLandLordName().equals(getLandLordName())
Comment